test/e2e/helpers.js

```diff
@@ -426,7 +426,7 @@ const importWrongSRPOnboardingFlow = async (driver, seedPhrase) => {
   );
 
   const warningText = 'Invalid Secret Recovery Phrase';
-  const warnings = await driver.findElements('.actionable-message__message');
+  const warnings = await driver.findElements('.banner-alert-text');
```

Comment:
To make this more specific to the import srp page could we prefix this className?
```suggestion
  const warnings = await driver.findElements('.import-srp__banner-alert-text');
```

Reply:
Done. Good Suggestion.